frontendUiTableImpl/src/main/java/com/virtuslab/gitmachete/frontend/ui/impl/table/EnhancedGraphTable.java

```diff
@@ -192,7 +196,8 @@ private void refreshModel(GitRepository gitRepository, List<String> skippedBranc
 
     if (!isMacheteFilePresent) {
       LOG.info("Machete file (${macheteFilePath}) is absent, so auto discover is running");
-      doAutomaticDiscover(macheteFilePath);
+      queueAutomaticDiscover(macheteFilePath, doOnUIThreadWhenReady);
```

Comment:
Sometimes (although veeery rarely) the model returned in `refreshGraphTableModel` method in `com.virtuslab.gitmachete.uitest.UITestSuite#discoverBranchLayout` (line 96: `int branchRowsCount = callJs("project.refreshGraphTableModel().getRowCount()");`) was the one from **before** the discovery running in the background was completed (and thus this model had zero rows, not seven).

Reply:
Maybe leave short comment about this fact in code